orderbook/openapi.yml

```diff
@@ -588,7 +588,7 @@ components:
       properties:
         errorType:
           type: string
-          enum: [InvalidSignature, WrongOwner, OrderNotFound]
+          enum: [InvalidSignature, WrongOwner, OrderNotFound, OrderAlreadyCancelled, OrderFullyExecuted, OrderExpired]
```

Comment:
ubernit: I think `AlreadyCancelled` can live without the term Order in front, but the other two (for some reason) make sense to have... I'm weird.

Reply:
I don't mind either way. Will update.